internal/security/anchore_client.go

```diff
@@ -48,6 +48,8 @@ type PolicyClient interface {
 
 type RegistryClient interface {
 	AddRegistry(ctx context.Context, registry Registry) error
+	GetRegistry(ctx context.Context, registryName string) ([]anchore.RegistryConfiguration, error)
+	UpdateRegistry(ctx context.Context, registry Registry) error
```

Comment:
As pointed out by @pepov , deletion is not implemented, which seems to be a bug to me.